### ISSUE TITLE
Issue with setup.py in Dataflow

### ISSUE THREAD

**Issue by raspstephan** [google-research/weatherbench2#99]
Running scripts on Dataflow requires passing a setup.py, see: https://weatherbench2.readthedocs.io/en/latest/beam-in-the-cloud.html

However, the gcp requirements are not part of the base requirements in setup.py. Simply using `pip install -e .[gcp]` locally does not seem to be enough to have the required packages installed for the workers. 

Is there a way to specify which requirements should be installed for the workers? If not, we might need to make the gcp requirements part of the base requirements.

**Comment by mahrsee1997** [comment]
In weather-tools we are explicitly specifying the requirements in setup.py (https://github.com/google/weather-tools/blob/main/setup.py#L17) .

**Comment by alxmrs** [comment]
I agree with Rahul. For this audience, having a pre-made Docker container is the way to go.
